src/registry_schemas/schemas/business.json

```diff
@@ -155,8 +155,8 @@
           "type": "string",
           "title": "The Tax_id Schema",
           "default": "",
-          "examples": ["123456789"],
-          "pattern": "^[0-9]{9}$"
+          "examples": ["123456789BC0001", "123456789"],
+          "pattern": "^[0-9]{9}[a-zA-z]{2}[0-9]{4}$|^[0-9]{9}$"
```

Comment:
Should we just allow uppercase codes?

Otherwise, regex looks good to me.

Reply:
(The UI does not enter Tax ID, so it's whatever the API can provide in the first place.)

Reply:
I am not expecting lowercase. @thorwolpert do you see any possibility to have lowercase? if not I will change it to `[A-Z]`

Reply:
I didn't find any taxId with lowercase in PROD. Removing lowercase.